src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -9,12 +9,27 @@
     <LangVersion>preview</LangVersion>
     <UserSecretsId>b8ffb8d3-b768-460b-ac1f-ef267c954c85</UserSecretsId>
     <PublishAot>true</PublishAot>
-    <EnableLogging Condition="$(Configuration.StartsWith('Debug'))">true</EnableLogging>
+    <EnableConfigurationBindingGenerator>false</EnableConfigurationBindingGenerator>
```

Comment:
Why are we disabling this?

Reply:
'Cos it's currently broken (it literally generates code with a compilation bug)